### ISSUE TITLE
Build failure on Windows

### ISSUE THREAD

**Issue by Jan-E** [jedisct1/libsodium-php#133]
@jedisct1 Even with a changed configure option (--with-sodium) builds on Windows fail now:
```
C:\php-sdk\php71dev>nmake php_sodium.dll

Microsoft (R) Program Maintenance Utility Version 14.00.24210.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        "cl.exe" /I "C:\php-sdk\win64build\include" /DHAVE_SODIUM_H=1 /D COMPILE_DL_SODIUM /D SODIUM_EXPORTS=1 /nologo /I . /I main /I Zend /I TSRM /I ext /D _WINDOWS /D ZEND_WIN32=1 /D PHP_WIN32=1 /D WIN32 /D _MBCS /W3 /FD /wd4996 /guard:cf /Zc:inline /Gw /MP /Zi /LD /MD /W3 /Ox /D NDebug /D NDEBUG /D ZEND_WIN32_FORCE_INLINE /GF /D ZEND_DEBUG=0 /D ZTS=1 /I "C:\php-sdk\win64build\include" /D FD_SETSIZE=256 /FoC:\php-sdk\php71dev\x64\Release_TS\ext\libsodium\ /FpC:\php-sdk\php71dev\x64\Release_TS\ext\libsodium\ /FRC:\php-sdk\php71dev\x64\Release_TS\ext\libsodium\ /FdC:\php-sdk\php71dev\x64\Release_TS\ext\libsodium\ /c ext\libsodium\libsodium.c
libsodium.c
ext\libsodium\libsodium.c(1683): warning C4018: '<': signed/unsigned mismatch
ext\libsodium\libsodium.c(1687): warning C4018: '<': signed/unsigned mismatch
ext\libsodium\libsodium.c(1725): warning C4018: '<': signed/unsigned mismatch
ext\libsodium\libsodium.c(1729): warning C4018: '<': signed/unsigned mismatch
ext\libsodium\libsodium.c(2899): error C2065: 'crypto_kdf_blake2b_CONTEXTBYTES': undeclared identifier
ext\libsodium\libsodium.c(2900): error C2065: 'crypto_kdf_blake2b_CONTEXTBYTES': undeclared identifier
ext\libsodium\libsodium.c(2905): warning C4293: '>>': shift count negative or too big, undefined behavior
ext\libsodium\libsodium.c(2906): warning C4293: '>>': shift count negative or too big, undefined behavior
ext\libsodium\libsodium.c(2907): warning C4293: '>>': shift count negative or too big, undefined behavior
ext\libsodium\libsodium.c(2908): warning C4293: '>>': shift count negative or too big, undefined behavior
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe"' : return code '0x2'
Stop.
```

**Comment by jedisct1** [comment]
Should be fixed in version 2.0.2, thanks!

Alternatively, use libsodium > 1.0.11.

**Comment by Jan-E** [comment]
Ack. Builds without warning with libsodium 1.0.13 and ext version 2.0.2. See for instance
https://phpdev.toolsforresearch.com/php-7.1.7-nts-Win32-VC14-x64.htm
